cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io",
+  "body": "Fixtures are a great way to mock data for responses to routes"
+}
```

Comment:
I think this, and the image of the failure below can be removed, no reason to have them in the repo.

Also we should add them to the `.gitignore` so they don't get accidentally committed again.